backend/tests/unit/apps/slack/shortcuts/report_content_test.py

```
@@ -0,0 +1,888 @@
+
+        post.assert_called_once_with("https://hooks.slack.test/r", "hi")
+
+    def test_post_ephemeral_url_swallows_request_errors(self, mocker):
+        """Test response_url failures are logged and ignored."""
+        mocker.patch(
+            "apps.slack.shortcuts.report_content.requests.post",
+            side_effect=requests.RequestException("boom"),
+        )
+        post_ephemeral_url("https://hooks.slack.test/r", "hi")
+
+    def test_open_modal_missing_message_ts(self, mocker):
```

Ruff (0.16.1)

[warning] Unused method argument: `mocker`

(ARG002)